components/NavigationBar/src/MDCNavigationBar.h

```diff
@@ -98,6 +98,12 @@ IB_DESIGNABLE
  */
 @property(nonatomic, strong, nullable) UIView *titleView;
 
+/**
+ The font applied to the title of navigation bar. This font is set through UILabel Font property.
+ Font size is enforced to 20.
```

Comment:
> Font size is enforced to 20.

Just to clarify, does this mean that no matter what UIFont I provide, it will resize the font to 20 points?

Reply:
Correct.

Reply:
I've filed https://www.pivotaltracker.com/story/show/156482550 to discuss this further. I also agree that the intention to fix to a specific font size is a bit surprising.